src/ProjectService.Business/Commands/Project/CreateProjectCommand.cs

```diff
@@ -35,38 +35,45 @@ public class CreateProjectCommand : ICreateProjectCommand
         private readonly IAccessValidator _accessValidator;
         private readonly ICreateProjectValidator _validator;
         private readonly ILogger<CreateProjectCommand> _logger;
-        private readonly IProjectInfoMapper _projectInfoMapper;
         private readonly IHttpContextAccessor _httpContextAccessor;
-        private readonly IRequestClient<IGetDepartmentRequest> _rcGetDepartment;
         private readonly IRequestClient<ICreateWorkspaceRequest> _rcCreateWorkspace;
         private readonly IRequestClient<ICheckUsersExistence> _rcCheckUsersExistence;
         private readonly IRequestClient<ICreateWorkTimeRequest> _rcCreateWorkTime;
+        private readonly IRequestClient<ICheckDepartmentsExistence> _rcCheckDepartmentsExistence;
 
-        private IGetDepartmentResponse GetDepartment(Guid departmentId, List<string> errors)
+        private List<Guid> CheckDepartmentExistence(Guid? departmentId, List<string> errors)
```

Comment:
check existence in validator, if request contains incorrect department id i think we have to throw validation exc, but you create project. I don't think this is good.

Reply:
fixed